tests/php/restricted_site_access.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+class RestrictedSiteAccessTests extends \PHPUnit\Framework\TestCase {
+	public function setUp() {
+		\WP_Mock::setUp();
+	}
+
+	public function tearDown() {
+		\WP_Mock::tearDown();
+	}
+
+	/**
+	 * Test that ip addresses pass or fail as expected when compared against an ip with a mask.
+	 *
+	 * @dataProvider ip_data_provider
+	 *
+	 * @param {string}   The ip address to test.
+	 * @param {string}   The ip plus mask to test against.
+	 * @param {@boolean} The expected result, success (true) or failure (false).
+	 */
+	public function testIpInRange( $ip_to_test, $ip_plus_mask, $expected )
+	{
+		$this->assertEquals( $expected, Restricted_Site_Access::ip_in_range( $ip_to_test, $ip_plus_mask ) );
+	}
+
+	public function ip_data_provider()
+	{
+		return [
```

Comment:
I wonder if we should use the same array syntax as the plugin and other tests to be consistent. Are there any concerns with using the shorthand syntax with this plugin? It has to support WP minimum requirements I believe.